src/coreclr/jit/compiler.h

```diff
@@ -8932,6 +8932,20 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 #endif // FEATURE_SIMD
 
 public:
+    // Similar to roundUpSIMDSize, but for General Purpose Registers (GPR)
+    unsigned int roundUpGPRSize(unsigned size)
+    {
```

Comment:
It would be better to have the `size <= REGSIZE_BYTES` assert in this function than in the callers.

Reply:
I am not sure it should care about the input size, e.g. for size 100 it will return 8 🤔 just like `roundUpSIMDSize` doesn't care about it either - caller can use this one to decide what kind of GPR reg it needs to process the given size, the only difference between roundUp and roundDown are the possability to overlap

Reply:
I see, it just caught my eye that it doesn't really round up for those cases